test/integration/dialects/mysql/connector-manager.test.js

```diff
@@ -15,19 +15,23 @@ if (Support.dialectIsMySQL()) {
         , self = this;
 
       return User.sync({force: true}).then(function() {
-        return User.create({username: 'user1'}).then(function() {
+        return User.count({group: ['id']}).then(function(count) {
+          expect(count.length).to.equal(0);
```

Comment:
Should be an integer.
[`count([options]) -> Promise.<Integer>](http://sequelize.readthedocs.org/en/latest/api/model/index.html?highlight=count#countoptions-promiseinteger)